src/libguac/recording.c

```diff
@@ -55,7 +55,7 @@ int guac_recording_open(const char* path,
 
     /* Attempt to open recording */
     int fd = open(basename,
-            O_CREAT | O_EXCL | O_WRONLY,
+            O_CREAT | O_WRONLY,
```

Comment:
What happens if the file is not empty? shouldn't we validate?

Reply:
the file is created by the idac right before the session begins with a uuid name so it is indeed empty

Reply:
it's an implicit dependency that future developers can easily break; suggest to make it explicit by checking that the file exists and is empty.

Reply:
if the file doesnt exist the file will be created. if it is not empty (which is very unlikely to happen), then it will fail to be validated upon processing when session ends